contracts/fixed_price.scilla

```diff
@@ -1106,6 +1123,26 @@ procedure Cancel(
   end
 end
 
+procedure BatchCancelOrder_(order: CancelOrderParam)  
+  match order with
+  | CancelOrderParam token_address token_id payment_token_address sale_price side =>
+    cur_owner <- contract_owner;
```

Comment:
@razill isn' t line 1129 to 1141 redundant? This has already been checked in the outer transition SetBatchCancel call? There is no need to recheck again if the contract is paused or not.